github/messages.go

```diff
@@ -122,6 +126,7 @@ func messageMAC(signature string) ([]byte, func() hash.Hash, error) {
 
 // ValidatePayload validates an incoming GitHub Webhook event request
 // and returns the (JSON) payload.
+// The payload can be of Content-Type application/json or application/x-www-form-urlencoded
```

Comment:
Nitpicks.

This sentence is missing a period at the end.

Also, I'd rephrase it as:

```
// The payload can have content type of "application/json" or "application/x-www-form-urlencoded".
```

`Content-Type` is the name of the header, but it's pretty natural to use the phrase "content type" or "type of content" in English. E.g., [`ProjectCardOptions.ContentType`](https://godoc.org/github.com/google/go-github/github#ProjectCardOptions.ContentType) is documented as "The type of content to associate with this card."